datafusion/datasource-parquet/src/file_format.rs

```diff
@@ -1576,3 +1715,164 @@ fn create_max_min_accs(
         .collect();
     (max_values, min_values)
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
Is it possible to add a real small parquet file that has this property to ensure everything is hooked up right? I worry this unit test might not entirely cover what is actually seen in the real ecosystem

Reply:
I'll look into this, thanks!

Reply:
if it is a small enough it can be like https://github.com/apache/datafusion/blob/main/datafusion/core/tests/data/timestamp_with_tz.parquet

Reply:
Sounds good, I'll do that today.